mingw-w64-wintoast/PKGBUILD

```diff
@@ -12,7 +12,7 @@ license=('MIT')
 
 options=('!strip')
 
-source=("${_realname}"::"git+https://github.com/git-for-windows/WinToast#tag=g4w-20171117-01")
+source=("${_realname}"::"git+https://github.com/git-for-windows/WinToast#tag=TODO_WHEN_PR_MERGED")
```

Comment:
This needs https://github.com/git-for-windows/WinToast/pull/2 to be merged first